vol_file_test_parallel.c

```diff
@@ -152,7 +152,7 @@ test_open_file(void)
     }
     END_MULTIPART;
 
-    TESTING_2("test cleanup")
+    TESTING_2("test H5Pclose")
```

Comment:
This isn't testing H5Pclose in particular. These messages refer to testing that resources can be cleaned up after a multi-part test so that the user knows whether testing failed during the test, or during cleanup of resources.

Reply:
I wrote that code and it's specifically for cleaning up files generated in the parallel file tests, which are separate from the `vol_test_parallel.h5` file that contains all the other tests like the dataset ones. Cleanup of resources _is_cleanup.

Reply:
What do you think the meaning of "cleanup" here from 1998?

```

/*-------------------------------------------------------------------------
 * Function:  h5_cleanup
 *
 * Purpose:  Cleanup temporary test files.
 *    base_name contains the list of test file names.
 *    The file access property list is also closed.
 *
 * Return:  Non-zero if cleanup actions were performed; zero otherwise.
 *
 * Programmer:  Albert Cheng
 *              May 28, 1998
 *
 *-------------------------------------------------------------------------
 */
int
h5_cleanup(const char *base_name[], hid_t fapl)
{
    int retval = 0;

    if (GetTestCleanup()) {
        /* Clean up files in base_name, and the FAPL */
        h5_clean_files(base_name, fapl);

        retval = 1;
    } /* end if */

    /* Restore the original error reporting routine */
    h5_restore_err();

    return retval;
} /* end h5_cleanup() */
```

Reply:
Nothing in "cleanup" refers specifically to files.

Reply:
Then, why do you say `clean up` here? 

```On the other hand, the parallel test does seem to be missing a call to H5Fdelete to clean up after itself```

Reply:
Because in that particular instance I _was_ referring specifically files.